### ISSUE TITLE
BBC: RURBAN/B-Stats-0.09.tar.gz fails since perl 5.31.9

### ISSUE THREAD

**Issue by eserte** [Perl/perl5#17765]
The test suite of B::Stats 0.09 fails since 5.31.9, see http://fast-matrix.cpantesters.org/?dist=B-Stats%200.09

A module bug ticket was written (https://rt.cpan.org/Ticket/Display.html?id=131963) and the problem was workarounded in 0.10, but Reini thinks that this is a perl regression, so probably worth a look.

**Comment by jkeenan** [comment]
Bisection:
```
perl Porting/bisect.pl --start=v5.31.8 --end=41fd77f86ddaa01f1a7635e15cbfd3ccb9a4643d --module=B::Stats
```
... points to:
```

4b004c43ef26ce175181122a7dfc3acd7dacb170 is the first bad commit
commit 4b004c43ef26ce175181122a7dfc3acd7dacb170
Author: Tony Cook <tony@develop-help.com>
Date:   Tue Jan 21 16:12:44 2020 +1100

    always treat undef in %INC as a failed require
    
    Previously require would check for the specific \&PL_sv_undef
    SV in %INC, this meant that if %INC was copied, or undef
    assigned to a member the entry would erroneously be treated as if
    a previous require of that file was successful.
    
    So check for SvOK() instead, with appropriate magic tests.
    
    fixes #17428
```
@tonycoz, can you take a look?

Thank you very much.
Jim Keenan

**Comment by tonycoz** [comment]
This is a bug in B::Stats, which is modifying the values of %INC, I've commented on the rt.cpan ticket at https://rt.cpan.org/Ticket/Display.html?id=131963#txn-1896311

Closing.
